scripts/vars.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export API_IMAGE="us-docker.pkg.dev/abcxyz-artifacts/docker-images/jvs-justification:0.0.4"
```

Comment:
Only one of these envvars is actually exported

Reply:
Yes, still playing around with these scripts. more updates are coming.